statsd/statsd.go

```diff
@@ -77,7 +77,7 @@ var (
 // A statsdWriter offers a standard interface regardless of the underlying
 // protocol. For now UDS and UPD writers are available.
 type statsdWriter interface {
-	Write(data []byte) error
+	Write(data []byte) (n int, err error)
```

Comment:
🍰 

This is a necessary fix for sure, the "problem" is it breaks the current implementation. All good to merge to master, we should probably tag a new major?

Reply:
Agreed, moved the milestone `2.0` and we can release accordingly